docs/changes.rst

```diff
@@ -135,66 +135,66 @@ Compatibility notes:
 
 - init: the --encryption argument is mandatory now (and there are several choices).
 - moved "borg migrate-to-repokey" to "borg key migrate-to-repokey".
-- "borg change-passphrase" is deprecated, use "borg key change-passphrase" instead.
+  "borg change-passphrase" is deprecated, use "borg key change-passphrase" instead.
```

Comment:
btw. those that I moved around I moved up/down for the impact I see for them, that's why I put the network, timestamp and hashindex fixes at the top of the bug fixes